package.json

```diff
@@ -49,6 +49,7 @@
   "homepage": "https://github.com/reaviz/reaflow#readme",
   "dependencies": {
     "@juggle/resize-observer": "^3.4.0",
+    "antd": "^5.22.0",
```

Comment:
Can you use `reablocks` tooltip? This projecty already uses it.

Reply:
I tried it but I have a problem when using component of reablocks in reaflow

I'm trying to import reablocks `Popover` into `Basic.stories.tsx`
Error screams at me right now: 
```
useTheme must be used within a ThemeProvider
```
I really don’t know how to solve this Error. using other third-party libraries to achieve custom tooltips doesn’t have this problem at all and can be achieved quickly.

Reply:
Take a look at how we did it in reaviz - https://github.com/reaviz/reaviz/blob/master/src/common/Tooltip/ChartTooltip.tsx#L41

Reply:
ok, I've changed `Node`'s tooltip props demo to use `reablocks`.

Added `tailwindcss`
Modify `postcss.config.cjs` to make Tailwind style take effect
Modify `.storybook/preview.tsx` to make `stories` use `reablocks` Theme